While this is done later (in uilib.pri), because libQt5UiTools is static,
this needs to be done before the load(qt_module) in order to be included in
mkspecs/modules/qt_lib_uitools.pri:QT.uitools.depends

--- origsrc/qttools-opensource-src-5.2.0/src/designer/src/uitools/uitools.pro	2013-12-08 11:10:14.000000000 -0600
+++ src/qttools-opensource-src-5.2.0/src/designer/src/uitools/uitools.pro	2014-01-06 18:45:07.187571400 -0600
@@ -1,6 +1,6 @@
 TARGET = QtUiTools
 CONFIG += static
-QT = core
+QT += widgets
 
 QMAKE_DOCS = $$PWD/doc/qtuitools.qdocconf
 
